index.html

```diff
@@ -6,9 +6,25 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Classwork/Practice</title>
     <link rel="stylesheet" href="css/bootstrap.min.css">    
-    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="css/styles.css">
 </head>
 <body>
-
+<div class="box container-fluid col-md-12 row yellow">[container-fluid]
+    <div class="container">
+        <div class="box header container col-md-12 row blue">[header]</div>
+        <div class="box nav container col-md-12 row green">[nav]</div>
+        <div class="row">
+            <div class="box content container col-md-9 red">[content]</div>
+            <div class="box aside container col-md-3 light-blue">[aside]</div>
+        </div> <!--Why does it poke out on the right?-->
+        <div>
```

Comment:
Try adding a class of `row` here in this empty div.

Reply:
It already has one. My comment is at the closing tag of the div, but I have "row" at the opening tag of it.

Reply:
Ahh yes I see that. I’ll give you an example of what I mean.

`<div class=“container”>`
       `<div class=“row”>`
              `<div class=“col-md-12 box some-other-class”>`
                       (Some content)
                `</div>`
           `</div>`
  `</div>`

Reply:
Dang it I was hoping it would be structured correctly but it didn’t take lol. But I hope you get what I mean!

Reply:
So you mean a container "contains" a row, and a row "contains" col's in increasingly nested divs. Is that correct?
I'm hoping that below is the proper nesting that you intended. At least that's what I understood you to mean.

<div class=“container”>
     <div class=“row”>
          <div class=“col-md-12 box some-other-class”>
             (Some content)
          </div>
     </div>
</div>

Reply:
Whoops - apparently you can't paste code in here, or it renders it, instead of displaying it. Ha!

Reply:
You can actually do it in your editor on the and branch. Git add., git etc... and will update the pr with the newest commit and show up here . It’s super cool.

Reply:
I just did that (yes, SUPER cool), but please don't worry about replying right now. I have to get to bed, and I'm sure you do too... :)